README.rst

```diff
@@ -14,7 +14,7 @@ A new octree-based game+engine using `Vulkan API <https://www.khronos.org/vulkan
 
 Our current octree demo ``v0.1-alpha.2`` can be downloaded `here <https://github.com/inexorgame/vulkan-renderer/releases>`__. Please send your logfiles to info@inexor.org.
 
-Please visit `inexor.org <https://inexor.org>`__ and join our `discord <https://discord.gg/acUW8k7>`__.
+Please visit `inexor.org <https://inexor.org>`__ and join our `discord <https://discord.gg/acUW8k7>`__ server.
```

Comment:
I would use this chance and write Discord with uppercase
```suggestion
Please visit `inexor.org <https://inexor.org>`__ and join our `Discord <https://discord.gg/acUW8k7>`__ server.
```

Reply:
Done.